packages/fresh_dio/lib/src/fresh.dart

```diff
@@ -84,15 +87,21 @@ class Fresh<T> extends Interceptor with FreshMixin<T> {
   }
 
   @override
-  Future<dynamic> onResponse(Response response) async {
+  Future<dynamic> onResponse(
+    Response response,
+    ResponseInterceptorHandler handler,
+  ) async {
     if (await token == null || !_shouldRefresh(response)) {
       return response;
     }
     return _tryRefresh(response);
```

Comment:
First of all, thanks for taking the time to make these changes! Have you verified that the example runs properly and refreshes/revokes correctly? I believe rather than returning in we should use the `handlers` in the various dio hooks.

Reply:
Thanks! You're 100% correct, I'll work on the required changes and update PR :)

Reply:
Hey, I believe I've updated your implementation to address these issues and plan to publish a stable version very soon 👍

Reply:
Awesome job! Apologies for not getting back to this sooner -- Love your speedy work! 💯 🚀